### DISCUSSION TITLE
How to sort a DataGrid on multiple columns from code

### DISCUSSION THREAD

**Discussion by wgraaf** [AvaloniaUI/Avalonia#14740] [Q&A]
A DataGrid can be sorted from code on a single column by calling `DataGridColumn.Sort()`.

Is there a way to sort on multiple columns from code?

**Comment by workgroupengineering** [comment]
You should use DataGridCollectionView.SortDescriptions https://github.com/AvaloniaUI/Avalonia/blob/d2023a94e8410f9a88314e3404158c9e311fef3c/samples/ControlCatalog/Pages/DataGridPage.xaml.cs#L34
